pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -0,0 +1,706 @@
+// Copyright (c) 2019 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Flow Statistics collection from IPtable Conntrack facility
+
+package zedrouter
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/eriknordmark/netlink"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
+	"github.com/lf-edge/eve/pkg/pillar/cast"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/lf-edge/eve/pkg/pillar/wrap"
+	"github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"syscall"
+)
+
+type flowStats struct {
+	SrcIP       net.IP
+	DstIP       net.IP
+	SrcPort     uint16
+	DstPort     uint16
+	Proto       uint8
+	SendPkts    uint64
+	SendBytes   uint64
+	RecvPkts    uint64
+	RecvBytes   uint64
+	TimeStart   uint64
+	TimeStop    uint64
+	TimeOut     uint32
+	aclNum      uint32
+	appNum      uint8
+	AppInitiate bool
+	IsTimeOut   bool
+	dbg1        int
+	dbg2        int
+}
+
+type aclAttr struct {
+	aclNum    uint32
+	tableName string
+	aclName   string
+	chainName string
+	action    string
+	vif       string
+	bridge    string
+	netUUID   uuid.UUID
+}
+
+type networkAttrs struct {
+	ipaclattr  map[int]map[int]aclAttr // appNum, ACLNum, acl attributes
+	appIPAddrs map[int][]net.IP        // appNum, IP addresses (may belong to diff bridges)
+	intfAddrs  []net.Addr              // device interface addresses
+	bnNet      map[string]aclAttr      // mainly need to range all the bridge interfaces
+	appNet     map[int]uuid.UUID       // max 256 apps
```

Comment:
App can be attached to multiple network instances. For ex. one interface of the app to local network instance and another interface to switch/mesh/cloud network instances.

Reply:
This is a mapping from what to what?